main_server/local_server/server_admin/c_comdef_admin_main_console.class.php

```diff
@@ -1879,6 +1879,87 @@ public function return_single_meeting_other_template()
                 case 'location_postal_code_1':
                 case 'location_nation':
                     break;
+                case 'comments':
+                    $ret .= '<div class="bmlt_admin_one_line_in_a_form clear_both">';
```

Comment:
I feel like all of these could be handled in one block using `$key`

Reply:
I feel like your right

Reply:
not sure how to do this while using localized strings and while still being able to allow custom fields, revert to default switch

Reply:
unless I check if the localized key exists

Reply:
Yes, if the key doesn’t exist then use what is in the db

Reply:
ok I did those things and tested with localized strings and without